pkg/openid4vp/openid4vp.go

```diff
@@ -85,19 +85,7 @@ func New(
 	documentLoader ld.DocumentLoader,
 	opts ...Opt,
 ) *Interaction {
-	client, activityLogger, metricsLogger, networkDocumentLoaderHTTPTimeout := processOpts(opts)
-
-	if documentLoader == nil {
```

Comment:
What happens if no document loader is passed in now?

Reply:
Created one with with embedded contexts

Reply:
The default document loader is set on the go mobile part.

Reply:
The Go API is also public, though. It can be used independently of the gomobile API, so it needs some default behaviour too.

Reply:
We pass a lot of parameters there are like didResolver, crypto, signatureVerifier that do not have default behavior too. We need some consistency. I am not sure that this is part of this optimization PR

Reply:
I see what you mean, good point! Just wondering - was there a technical reason you removed the default behaviour that was already there, or was it just more for consistency? (Either way, it's OK by me)

Reply:
On a side note - we should technically be returning errors and/or have some default behaviour if those other required params aren't passed in...

Reply:
I moved it to mobile wrappers, because for some code default document loader creation was already done on mobile wrappers. It should be consistent.